### PR TITLE
adding login page

### LINKED ISSUE #14: adding changes in styles
changing CSS and adding a logIn page

### PR DESCRIPTION
**Pull request by mariyasada**
adding login page feature to E-commerce
login

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -4,20 +4,22 @@ import MockMan from "mockman-js";
...
 import { ProductPage } from "./pages/ProductPage/ProductPage";
-import { SingleProductDetail } from "./pages/singleProductDetail/singleproductDetail";
+import { LogInPage } from "./pages/LogIN/LogInPage";
+
 
 
 
 function App() {
   return (
     // <div className="App">
```
Comment (t007rushi) [inline src/App.js]: remove unwanted comments

File: src/App.js [hunk 2]
```diff
@@ -4,20 +4,22 @@ import MockMan from "mockman-js";
 import {Routes,Route} from "react-router";
 import Home from "./pages/Home/Home";
 import { ProductPage } from "./pages/ProductPage/ProductPage";
-import { SingleProductDetail } from "./pages/singleProductDetail/singleproductDetail";
+import { LogInPage } from "./pages/LogIN/LogInPage";
```
Comment (t007rushi) [inline src/App.js:7]: as we go further there may be many component Pages to import so make it online using an ```index.js``` file instead

File: src/pages/LogIN/LogInPage.jsx [hunk 3]
```diff
@@ -0,0 +1,78 @@
+import React from 'react'
+import "../../components/ProductCard/Productcard.css";
+import { FaEyeSlash } from "react-icons/fa";
+import { Link } from 'react-router-dom';
+import "../LogIN/LogInPage.css";
+
+ export const LogInPage = () => {
```
Comment (t007rushi) [inline src/pages/LogIN/LogInPage.jsx:7]: some minor changes CSS can be written to center this component to page for all sizes

**Review by t007rushi — COMMENTED** [review]
Add favicon, change the title to Ecom name

File: src/App.js (hunk 1 above)
Comment (mariyasada) [reply src/App.js]: done

File: src/App.js (hunk 2 above)
Comment (mariyasada) [reply src/App.js:7]: done

File: src/pages/LogIN/LogInPage.jsx (hunk 3 above)
Comment (mariyasada) [reply src/pages/LogIN/LogInPage.jsx:7]: if i gave it parent div to center property it goes above the header that's why i gave a margin-left

**Review by Bhakti199 — APPROVED** [review]
I like that :
You have written code very well
proper naming conventions
you have maintain folder structure very well.
This PR is good to merge.

**Review by t007rushi — APPROVED** [review]

**Pull request merged**